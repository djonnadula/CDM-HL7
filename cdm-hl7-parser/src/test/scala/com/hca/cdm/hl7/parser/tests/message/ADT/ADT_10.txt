MSH|^~\&|MT6_FRNK|FRNK|DBM||20151221130800-0600||ADT^A40|MT6_FRNK_ADT_1129610.|D|2.5||KYB|||
EVN|A40|201512211308|||QBE8554^Hammock^Brittany|201512211302
PID|1||K000002505^^^^MR^FRNK~302-21-2015^^^^SS^FRNK~A00002475^^^^HUB^FRNK|A1-B20151218101601316^^^^URN^FRNK|Osborn^Norman^^^^^L~Goblin^Green^^^^^M||19840220|M|||123 Marvel Street^^Franklin^TN^37064^^P^^TN187||(615)555-6261|(502)875-7272||W||FR0000121533|302-21-2015
MRG|||K000002506^^^^MR
PV1|1|P|FR.LAB||||DR.ATND01^Attending01^Provider^^^^^^^^^^XX|||||||||||POV||05|||||||||||||||||||FRNK||PRE|||201512210000
PV2|||Labwork|||||||||||||||||||||||||||||||||N
ROL|1|AD|AT|DR.ATND01^Attending01^Provider^^^^^^^^^^XX
ROL|2|AD|PP|DR.PCP01^PrimaryCare01^Provider^^^^^^^^^^XX
DG1|1||D64.9^ANEMIA, UNSPECIFIED^I10|||Other
DRG|33