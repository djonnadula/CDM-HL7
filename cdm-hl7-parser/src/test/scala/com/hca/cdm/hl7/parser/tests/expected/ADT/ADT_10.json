{
  "0000.COMN": {
    "sending_facility": "FRNK",
    "message_control_id": "MT6_FRNK_ADT_1129610.",
    "medical_record_num": "K000002505",
    "medical_record_urn": "A1-B20151218101601316",
    "patient_account_num": "FR0000121533"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "MT6_FRNK",
    "sending_facility": "FRNK",
    "receiving_application": "DBM",
    "date_time_of_message": "20151221130800-0600",
    "message_type": {
      "message_code": "ADT",
      "trigger_event": "A40"
    },
    "message_control_id": "MT6_FRNK_ADT_1129610.",
    "processing_id": "D",
    "version_id": "2.5",
    "continuation_pointer": "KYB"
  },
  "0002.EVN": {
    "event_type_code": "A40",
    "recorded_date_time": "201512211308",
    "operator_id": {
      "person_id": "QBE8554",
      "family_name": "Hammock",
      "given_name": "Brittany"
    },
    "event_occured": "201512211302"
  },
  "0003.PID": {
    "set_id": "1",
    "patient_id_list": [
      {
        "id_num": "K000002505",
        "id_type_code": "MR",
        "assigning_facility": "FRNK"
      },
      {
        "id_num": "999-99-9999",
        "id_type_code": "SS",
        "assigning_facility": "FRNK"
      },
      {
        "id_num": "A00002475",
        "id_type_code": "HUB",
        "assigning_facility": "FRNK"
      }
    ],
    "alternate_patient_id": {
      "id_num": "A1-B20151218101601316",
      "id_type_code": "URN",
      "assigning_facility": "FRNK"
    },
    "patient_name": [
      {
        "family_name": "Osborn",
        "given_name": "Norman",
        "name_type_code": "L"
      },
      {
        "family_name": "Goblin",
        "given_name": "Green",
        "name_type_code": "M"
      }
    ],
    "date_time_of_birth": "19840220",
    "administrative_sex": "M",
    "patient_address": {
      "street_address": "123 Marvel Street",
      "city": "Franklin",
      "state_or_province": "TN",
      "zip_or_postal_code": "37064",
      "address_type": "P",
      "county_parish_code": "TN187"
    },
    "phone_num_home": "(615)555-6261",
    "phone_num_business": "(502)875-7272",
    "marital_status": "W",
    "patient_account_num": "FR0000121533",
    "ssn_num_patient": "302-21-2015"
  },
  "0004.MRG": {
    "prior_patient_account_num": {
      "id_num": "K000002506",
      "id_type_code": "MR"
    }
  },
  "0005.PV1": {
    "set_id": "1",
    "patient_class": "P",
    "assigned_patient_location": "FR.LAB",
    "attending_doctor": {
      "person_id": "DR.ATND01",
      "family_name": "Attending01",
      "given_name": "Provider",
      "id_type_code": "XX"
    },
    "patient_type": "POV",
    "financial_class": "05",
    "servicing_facility": "FRNK",
    "account_status": "PRE",
    "admit_date_time": "201512210000"
  },
  "0006.PV2": {
    "admit_reason": "Labwork",
    "newborn_baby_indicator": "N"
  },
  "0007.ROL": {
    "role_instance_id": "1",
    "action_code": "AD",
    "role_rol": "AT",
    "role_person": {
      "person_id": "DR.ATND01",
      "family_name": "Attending01",
      "given_name": "Provider",
      "id_type_code": "XX"
    }
  },
  "0008.ROL": {
    "role_instance_id": "2",
    "action_code": "AD",
    "role_rol": "PP",
    "role_person": {
      "person_id": "DR.PCP01",
      "family_name": "PrimaryCare01",
      "given_name": "Provider",
      "id_type_code": "XX"
    }
  },
  "0009.DG1": {
    "set_id": "1",
    "diagnosis_code": {
      "id": "D64.9",
      "text": "ANEMIA, UNSPECIFIED",
      "name_of_coding_system": "I10"
    },
    "diagnosis_type": "Other"
  },
  "0010.DRG": {
    "diagnostic_related_group": "33"
  }
}